src/Transfer/EzPlatform/Repository/Manager/ContentManager.php

```diff
@@ -279,4 +284,25 @@ private function assignStructFieldValues(ContentObject $object, $struct)
             $struct->setField($key, $value);
         }
     }
+
+    /**
+     * @param ContentObject $object
+     * @param Location      $location
+     */
+    private function ensureNotTrashed(ContentObject $object)
+    {
+        $query = new Query();
+        $query->filter = new Criterion\ContentId($object->getContentInfo()->id);
+        $trash = $this->repository->getTrashService()->findTrashItems($query);
+        if ($trash->count > 0) {
+            /** @var TrashItem $trashItem */
+            $trashItem = $trash->items[0];
+            $parentLocation = $this->repository->getLocationService()->loadLocation($trashItem->parentLocationId);
+            $this->repository->getTrashService()->recover($trashItem, $parentLocation);
+            if ($this->logger) {
```

Comment:
I tried that accually. Same result (still not covered). Committing it temporarily to get it confirmed.